frontend/src/metabase/collections/components/ActionMenu/ActionMenu.tsx

```diff
@@ -88,12 +98,12 @@ function ActionMenu({
         isBookmarked={isBookmarked}
         isPreviewShown={isPreviewShown(item)}
         isPreviewAvailable={isFullyParametrized(item)}
-        onPin={collection.can_write && item.setPinned ? handlePin : null}
-        onMove={collection.can_write && item.setCollection ? handleMove : null}
+        onPin={canPin && item.setPinned ? handlePin : null}
```

Comment:
since `canPin` already tests that`item.setPinned != null`, it should be fine to remove `item.setPinned` from this condition what do you think?

```suggestion
        onPin={canPin ? handlePin : null}
```

Reply:
Fixed